### PR TITLE
Init setup frontend

### PR DESCRIPTION
**Pull request by Solodkih**

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 from django.contrib import admin
 from . import views 
 from django.urls import path
+from django.conf.urls.static import static
```
Comment (porlov) [inline backend/backend/urls.py]: Why do we need these unused imports?

File: backend/backend/settings.py [hunk 2]
```diff
@@ -126,4 +124,13 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
+STATICFILES_FINDERS = [
```
Comment (porlov) [inline backend/backend/settings.py]: Please do all backend changes in the backend initial setup PR

File: docker-compose.yml [hunk 3]
```diff
@@ -6,7 +6,6 @@ volumes:
 services:
   db:
     image: postgres:11
-    restart: always
```
Comment (porlov) [inline docker-compose.yml]: the same for this, in another PR

File: frontend/.eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: ['react', 'jest'],
+  rules: {
+    'import/no-unresolved': 'off',
+    'import/prefer-default-export': 'warn',
+    'react/prop-types': 'off',
```
Comment (porlov) [inline frontend/.eslintrc.js]: I think we should not disable it

File: frontend/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+//const webpack = require('webpack');
```
Comment (porlov) [inline frontend/webpack.config.js]: unused code

File: frontend/webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const BUILD_DIR = path.resolve(__dirname, 'public/dist');
```
Comment (porlov) [inline frontend/webpack.config.js]: let's rename to `DIST_DIR` and `SCR_DIR`

File: backend/.gitignore [hunk 7]
```diff
@@ -0,0 +1 @@
+python-virtual-environments/
```
Comment (porlov) [inline backend/.gitignore]: These changes are already in backend PR?

File: frontend/babel.config.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (porlov) [inline frontend/babel.config.js]: We don't need babel, all modern browsers support ES6 features

File: frontend/.prettierrc.yaml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+printWidth: 100
```
Comment (porlov) [inline frontend/.prettierrc.yaml]: Why do we need prettier? We already have such rules in eslint, I think. And eslint has auto-fixer that can fix such errors automatically

File: frontend/package.json [hunk 10]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "frontend",
```
Comment (porlov) [inline frontend/package.json]: please use project name

File: frontend/src/App/index.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+
+function App() {
+  return (
+    <div className='container'>
```
Comment (porlov) [inline frontend/src/App/index.js]: Use react-bootstrap `Container`

File: frontend/src/index.js [hunk 12]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import './index.scss';
+import { render } from 'react-dom';
```
Comment (porlov) [inline frontend/src/index.js]: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

File: frontend/babel.config.js (hunk 8 above)
Comment (Solodkih) [reply frontend/babel.config.js]: Bable jsx parsing, without Bable webpack don't understand jsx.

File: frontend/src/index.js (hunk 12 above)
Comment (Solodkih) [reply frontend/src/index.js]: I am install this plugin, but it don`t see errors on this place. There are no errors with the default settings.

File: frontend/src/index.js [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+import App from './App';
+
+render(<div><App /></div>, document.getElementById('app'));
```
Comment (porlov) [inline frontend/src/index.js]: We can render just as: `render(<App />, document.getElementById('app'));`

**Pull request merged**